composer.json

```diff
@@ -1,8 +1,18 @@
 {
-    "name": "omines/datatables-bundle",
+    "name": "fluxter/datatables-bundle",
```

Comment:
You're welcome to fork the bundle but please don't push that upstream   😉

Reply:
whoops, shall i change that so it can be merged?  
I did some more improvements and added the javascript into the js file  
https://github.com/Fluxter/datatables-bundle/commit/8202b844fca64e7ddeceebb9164c4071f7d5956f

Reply:
whoops, shall that be reverted so it can be merged?  
I did some more improvements in my fork tho, to have the js seperated https://github.com/Fluxter/datatables-bundle/commit/8202b844fca64e7ddeceebb9164c4071f7d5956f

Reply:
I'll review the rest once the first line is no longer a definite merge-stopper   😆